lib/web3.js

```diff
@@ -36,11 +36,14 @@ var c = require('./utils/config');
 
 /// @returns an array of objects describing web3 api methods
 var web3Methods = function () {
-    return [
-    { name: 'sha3', call: 'web3_sha3' }
-    ];
+    return [{
+        name: 'sha3',
+        call: 'web3_sha3'
+    }];
 };
 
+utils.phantomjsFix();
```

Comment:
currently phantomjs does not support Function.prototype.bind and that creates a bunch of issues. There is a link in the comment of the method with all the details.
This is *currently* required to allow testing with PhantomJS. PhantomJS is IMO the best option to test and develop because this is available on all plateforms and runs headless.

So I think this is worth having this fix in.